CondCore/CondDB/interface/Types.h

```diff
@@ -72,7 +72,7 @@ namespace cond {
     std::string name;
     Iov_t lastInterval;
     size_t size = 0;
-    bool isEmpty() const { return lastInterval.since != time::MAX_VAL; }
+    bool isEmpty() const { return lastInterval.since == time::MAX_VAL; }
```

Comment:
@ggovi this changes completely the logic of `isEmpty()` method, is this intended? (i.e. is it a bug-fix?)

Reply:
Thanks for the explanation Giacomo! then I agree with the change!